x-pack/platform/plugins/shared/streams_app/public/components/data_management/schema_editor/schema_changes_review_modal.tsx

```diff
@@ -90,25 +91,30 @@ export function SchemaChangesReviewModal({
           name: field.name,
         }));
 
-      const simulationResults = await streamsRepositoryClient.fetch(
-        'POST /internal/streams/{name}/schema/fields_simulation',
-        {
-          signal,
-          params: {
-            path: { name: stream },
-            body: {
-              field_definitions: mappedFields,
+      try {
+        const simulationResults = await streamsRepositoryClient.fetch(
+          'POST /internal/streams/{name}/schema/fields_simulation',
```

Comment:
I've hit timeouts when selecting ~10 fields. @flash1293 @tonyghiani is that a known issue ? do we have improvements planned or should we try to improve in this change ?
We could throttle and send sequential batches in the UI otherwise

Reply:
Hey Kevin, could you add a recording of how the UI is currently blocked in this scenario please?

Reply:
I'm not seeing that issue anymore after nuking my env, so I'm not sure if that was a bug I introduced or something else. I'll still try to reproduce